### DISCUSSION TITLE
My old username appears in current commits

### DISCUSSION THREAD

**Discussion by vangogiel** [community/community#29732] [Pull Requests] [answered]
Long time ago I had an account called n0rmanUK and changed the name to vangogiel. I created a pull request in a public repo and turns out the commit is signed by n0rmanUK. How is this even possible?

**Comment by radar-blip** [comment]
Sounds like you were really commited.

**Reply by strong-728** [reply]
I think you have used the more than 2 accounts on github. so you have to delete the account you dont want really in windows credentials manager.
After this, you can make push, pull, PR with account really you want.
Good luck with this answer. :)

**Comment by vangogiel** [comment, marked as answer]
😂 very good answer

I actually found out. 
In git, locally I have changed my email address that I use to commit with. That email address is assigned to my old account. 

Therefore, I did not change my account name from n0rmanUK to vangogiel. My vangogiel account cross references commits to the old account, because I used a wrong email in the local commit.

I have tried changing my local email address back and commit history is clean in GitHub. 

I am just wondering if I were to use somebody else's email address would the commit appear with their name under my Pull Request?

**Reply by radar-blip** [reply]
I dont know, you could always try I suppose. Dont know if thats a good idea though. 
Anyway, glad you figured evreything out!
